lib/logicware.rb

```diff
@@ -2,5 +2,6 @@
 
 module Logicware
   class Error < StandardError; end
-  # Your code goes here...
+
+  autoload :JWT, 'logicware/rack/jwt'
```

Comment:
use static requires

Reply:
here we can remove this and the developer will write himself the require

Reply:
create a file lib/logicware/rack.rb this is the file we would include if we need the middleware